### PR TITLE
Update to the latest version of the TDLib

### PR DESCRIPTION
**Pull request by vl-dev**
Hello,

the proposed change updates the library to the newest version of TDLib and fixes the reason why the `make generate-code` command was failing for this version.

I am happy to discuss this in more detail.

### CODE REVIEW

**Comment by zelenin** [comment]
I only support tagged versions

**Comment by vl-dev** [comment]
@zelenin Thank you for your response. Would you be open to discussing this? There is a lot of new functionality since the last version. I have seen people using patch versions as well: https://github.com/tdlib/td/issues/2215

Even if you are not open to this, I think that the following change could be beneficial as it fixes the generator on the latest version and doesn't break anything in the past. I can do it in a separate PR: https://github.com/vl-dev/go-tdlib/blob/3b759f54765c4d3b4ee4bcdfa63564b7e6e4dc41/tlparser/parser.go#L93-L111

**Comment by zelenin** [comment]
ok. let's try. I will now comment this PR

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-TAG := v1.8.0
+TAG := master
```
Comment (zelenin) [inline Makefile:1]: use a fixed commit

File: README.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
 # go-tdlib
 
-Go wrapper for [TDLib (Telegram Database Library)](https://github.com/tdlib/td) with full support of TDLib v1.8.0
+Go wrapper for [TDLib (Telegram Database Library)](https://github.com/tdlib/td) with full support of the newest TDLib (as of 2023/09/19)
```
Comment (zelenin) [inline README.md:3]: use a fixed commit and iso date

**Comment by zelenin** [comment]
please update and I will accept it

**Comment by vl-dev** [comment]
@zelenin Thank you for reconsidering. Your changes have been incorporated into the code

**Comment by zelenin** [comment]
thx for PR

**Pull request merged**